src/ducks/GainersAndLosers/GainersAndLosersTable.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import ReactTable from 'react-table'
+import { graphql } from 'react-apollo'
+import getColumns from './gainers-and-losers-table-columns'
+import { TOP_SOCIAL_GAINERS_LOSERS_QUERY } from './gainersLosersQuery'
+import { getTimeRangeByDuration } from '../../utils/utils'
+import 'react-table/react-table.css'
+
+const GainersLosersTable = ({
+  isLoading,
+  timeWindow,
+  topSocialGainersLosers,
+  error
+}) => (
+  <ReactTable
+    loading={isLoading}
+    showPaginationTop={false}
+    showPaginationBottom
+    defaultPageSize={10}
+    getTheadThProps={() => ({ style: { textAlign: 'center' } })}
+    getTrProps={() => ({ style: { textAlign: 'center' } })}
+    getTdProps={() => ({
```

Comment:
**[LOC 20, 21, 22, 29]** Unnecessary closures which are created on every rerender. Should be moved out and created as constants